ci/test/add

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -ex
+
+source ./common
+init_test_repo
+cd $test_repo_path
+
+popper init
+
+popper add cplee/github-actions-demo@master
+test -f /tmp/mypaper/.github/actions/jshint/Dockerfile
+test -f /tmp/mypaper/main.workflow
+
```

Comment:
might be preferable to remove the workflow file and `.github` folder after each test. Or create a new subfolder for each distinct test so that we guard against the possibility of `test -f ...` being executed against the previous execution of `popper add`.

Also, we could test with other workflows, e.g. the example ones in https://github.com/actions

Reply:
@ivotron Updated the tests. Thanks